### ISSUE TITLE
foreign export causing: java.lang.ClassFormatError: Duplicate field name&signature in class file

### ISSUE THREAD

**Issue by jrciii** [typelead/eta#550]
Using this project on Windows: https://github.com/jrciii/gradleEtaLibraryFromJava

I get the following:

```
>gradlew cleanEta compileEta
>gradlew runScratch
:compileJava
:processResources UP-TO-DATE
:classes
:runtimeEta
:runScratch
Exception in thread "main" java.lang.ClassFormatError: Duplicate field name&signature in class file scratch/scratch/Scratch
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at eta.scratch.Scratch.dude(Unknown Source)
        at com.jrciii.Scratchy.main(Scratchy.java:7)
:runScratch FAILED
```

The culprit in the scratch.scratch.Scratch class:
```
package scratch.scratch;

import ...

public class Scratch {
    **private static Closure dude;
    private static Closure dude;**
    private static Closure dude1;
    private static Closure $Lr34Slvl;

    **public static Closure dude() {
        if(dude == null) {
            dude = new Scratch.dude();
        }

        return dude;
    }

    public static Closure dude() {
        if(dude == null) {
            dude = new Scratch.dude();
        }

        return dude;
    }**

    public static Closure dude1() {
...
```
My .hs file src/main/eta/Scratch/Scratch.hs:
```
module Scratch.Scratch where

dude :: IO ()
dude = putStrLn "Omg"

foreign export java "@static eta.scratch.Scratch.dude" dude :: IO ()
```

**Comment by rahulmutt** [comment]
This is a duplicate of #547. I'll cut a release and close this once it's done.

**Comment by rahulmutt** [comment]
Just released 0.0.9b5 that contains the fix for this. Closing.
